cpp/autosar/test/rules/M8-5-2/test.cpp

```diff
@@ -55,4 +55,13 @@ void test() {
   Bar b2{{0}};      // NON_COMPLIANT - missing explicit init, nested zero init
   StructNested n{}; // COMPLIANT
   StructNested n1 = {}; // COMPLIANT
-}
\ No newline at end of file
+}
+
+#include <initializer_list>
+template <class T> bool all_of(std::initializer_list<T>);
+
+template <typename... Args> constexpr bool all_of(Args... args) noexcept {
+  return all_of({args...});
```

Comment:
Test case is missing `COMPLIANT - reason..` to document is compliant.

Reply:
Resolved.